app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -13,7 +13,8 @@
         return $(e.target).val();
       }
 
-      return $('[name="order[payments_attributes][][payment_method_id]').val();
```

Comment:
I would rather leave it as one-liner but close the `"` at the end of attribute rule. Cause it's missing there. ;-)